### PR TITLE
chore: use dockerhub instead of harbor

### PR DESCRIPTION
**Pull request by fryorcraken**

### CODE REVIEW

**Comment by fryorcraken** [comment]
I overrode approval process due to DOS of the harbor

File: register_rln.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ if test -n "${ETH_CLIENT_ADDRESS}"; then
   exit 1
 fi
 
-docker run -v "$(pwd)/keystore":/keystore/:Z harbor.status.im/wakuorg/nwaku:v0.31.0 generateRlnKeystore \
+docker run -v "$(pwd)/keystore":/keystore/:Z wakuorg/nwaku:v0.31.0 generateRlnKeystore \
```
Comment (chaitanyaprem) [inline register_rln.sh:23]: why 0.31.0, shouldn't this also be 0.32.0?

**Review by chaitanyaprem — APPROVED** [review]
LGTM, one minor comment

File: register_rln.sh (hunk 1 above)
Comment (fryorcraken) [reply register_rln.sh:23]: ah, I prefer not to touch it and break things :)

**Pull request merged**